packages/cardano-services-client/package.json

```diff
@@ -56,7 +56,9 @@
     "typescript": "^4.7.4"
   },
   "dependencies": {
+    "@blockfrost/blockfrost-js": "^5.7.0",
```

Comment:
It seems that we're using only [isBlockfrostErrorResponse](https://github.com/blockfrost/blockfrost-js/blob/a452d9b7500db1fea9bcce65ed783e5afca67bff/src/utils/errors.ts#L59) which imho doesn't justify the prod dependency. Maybe we should copy over this function and make it a `devDependency`